app/docs/0.1.1beta-2/cli.md

```diff
@@ -0,0 +1,132 @@
+
+```bash
+$ kong start
+```
+
+> **This command deserves a little more explanations about what happens when it's being run:**
+>
+1. First, if no configuration was provided as an argument (as in our example above) it will simply try to load a configuration at `/etc/kong/kong.yml`.
+>
+2. If you did not put a configuration at `/etc/kong/kong.yml`, you will be fine! Kong will load a default configuration.
+>
+3. Kong will try to connect to your configured datastore (most likely your Cassandra instance). If the connection is successful, Kong will prepare your database and make itself at home.
+>
+4. The NGiNX configuration specified in Kong's configuration (`nginx` property) will be used to spawn NGiNX workers, and those will use the working directory specified in your configuration (`nginx_working_directory`).
+
+If everything went well, you should see a successful message (`[OK] Started`).
+
+##### Specifying a configuration
+
+To override the default configuration, either place one at `/etc/kong/kong.yml`, or specify a file as an argument, like this:
+
+```bash
+$ kong start -c <path_to_config>
+```
+
+##### Congrats!
+
+Kong is now running and listening on two ports, which are by default:
+
+- `8000`, that will be used to process the API requests.
+- `8001`, called admin port, provides the Kong's internal RESTful API that you can use to operate Kong, and should be private and firewalled.
+
+---
+
+## stop
+
+This command executes a fast shutdown of Kong. It is actually a wrapper aroung the NGiNX `stop` signal:
+
+```bash
+$ kong stop
+```
+
+In order to stop an instance, the CLI needs to know what is the working directory of your process. If you started Kong with a certain configuration file, you must stop it with the same one. Thus, the stop command also accepts a configuration option:
+
+```bash
+$ kong stop -c <path_to_config>
+```
+
+If Kong stopped successfully, you should see a successful message (`[OK] Stopped`).
+
+> For more informations regarding the NGiNX signals, consult their [documentation][nginx-signals].
+
+---
+
+## quit
+
+`quit` performs a graceful shutdown of Kong. Like `stop`, it is a wrapper arround the NGiNX `quit` signal:
+
+```bash
+$ kong quit
+```
+
+And for the same reasons as `stop`, it also accepts a configuration option:
+
+```bash
+$ kong quit -c <path_to_config>
+```
+
+> For more informations regarding the NGiNX signals, consult their [documentation][nginx-signals].
+
+---
+
+## restart
+
+This command simply sends Kong a `stop` signal, followed by a `start` signal. If Kong was not running prior to the command, it will simply start it:
+
+```bash
+$ kong restart [-c path_to_config]
+```
+
+---
+
+## reload
+
+Very handy, this command will allow you to change your configuration at runtime, without any downtime. It takes advantage of NGiNX's [reload](http://wiki.nginx.org/CommandLine#Loading_a_New_Configuration_Using_Signals) signal to spawn new workers with the new configuration, while killing the old ones.
```

Comment:
- Not sure about the `very handy` statement, is it required?
- Avoid repetitive usage of `This command`, instead do something like so:

  Reloads the NGINX configuration at runtime and avoids potential downtime by leveraging the NGINX [reload](http://wiki.nginx.org/CommandLine#Loading_a_New_Configuration_Using_Signals) signal.